client/editor/map_editor.cpp

```diff
@@ -7,12 +7,13 @@
 // Client                     ::Where Needed::
 #include "citydlg.h"        //map_editor::showEvent
 #include "client_main.h"    //map_editor::check_open
+#include "editor.h"         //map_editor::showEvent
 #include "fc_client.h"      //map_editor::check_open
 #include "minimap_panel.h"  //map_editor::showEvent
 #include "page_game.h"      //map_editor::showEvent
 #include "views/view_map.h" //map_editor::showEvent
 
-// common
+// Common
```

Comment:
This is the folder name
```suggestion
// common
```